### PR TITLE
Chapter 4 & 5

### PR DESCRIPTION
**Pull request by rashidfirhat98**

### CODE REVIEW

**Comment by rashidfirhat98** [comment]
the previous commit appear after i rebase with master. Im not sure how to solve this

**Comment by rashidfirhat98** [comment]
ready

**Review by saobv-2223 — COMMENTED** [review]
Your PR is having 5 commits, merge to 1 commit please

**Comment by rashidfirhat98** [comment]
ready

File: app/assets/stylesheets/application.scss [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+ * compiled file so the styles you add here take precedence over styles defined in any other CSS/SCSS
+ * files in this directory. Styles in this file should be added after the last require_* statement.
+ * It is generally better to create a new file per style scope.
+ *
+ *= require_tree .
+ *= require_self
+ */
+ @import 'bootstrap-sprockets';
+ @import 'bootstrap';
+
```
Comment (manhnt-0960) [inline app/assets/stylesheets/application.scss]: add new line at end of file

File: app/views/layouts/application.html.erb [hunk 2]
```diff
@@ -7,9 +7,13 @@
...
     <%= javascript_pack_tag "application", "data-turbolinks-track": "reload" %>
+    <%= render 'layouts/shim' %>
   </head>
-
   <body>
-    <%= yield %>
+  <%= render "layouts/header"%>  
+  <div class="container">
+      <%= yield %>
+      <%= render 'layouts/footer' %>
```
Comment (manhnt-0960) [inline app/views/layouts/application.html.erb]: please use `"` instead of `'`

File: app/views/static_pages/contact.html.erb [hunk 3]
```diff
@@ -0,0 +1,7 @@
+<% provide(:title, 'Contact') %>
+
+<h1>Contact</h1>
+<p>
+  Contact the Ruby on Rails Tutorial about the sample app at the
+  <%= link_to "contact page", Settings.links.rails_tutorial.contact_page %>.
+</p>
```
Comment (manhnt-0960) [inline app/views/static_pages/contact.html.erb]: end line

File: config/routes.rb [hunk 4]
```diff
@@ -1,6 +1,9 @@
 Rails.application.routes.draw do
   root "static_pages#home"
-  get "static_pages/help"
-  get 'static_pages/about'
-  # For details on the DSL available within this file, see https://guides.rubyonrails.org/routing.html
-end
+  
+  get   "/home", to: "static_pages#home"
```
Comment (manhnt-0960) [inline config/routes.rb]:
```suggestion
  get "/home", to: "static_pages#home"
```

**Comment by rashidfirhat98** [comment]
ready

File: config/routes.rb (hunk 4 above)
Comment (rashidfirhat98) [reply config/routes.rb]: done

**Comment by rashidfirhat98** [comment]
ready

**Comment by manhnt-0960** [comment]
merged

**Pull request merged**